src/components/billing/contact/ContactUs.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Modal, Form, Input } from 'antd';
+import FormItemLabel from "../../form-item-label/FormItemLabel"
+
+const ContactUs = (props) => {
+    const { getFieldDecorator } = props.form;
+
+    return(
+    <Modal
+    title="Contact us"
+    okText="Submit"
+    visible={true}
+    >
+        <Form layout="vertical">
+            <FormItemLabel name="Subject" />
+            <Form.Item>
+                {getFieldDecorator('subject', {
+                    rules: [{ required: true, message: 'Please provide a subject' }]
```

Comment:
Take defaultValue here and pass the appropriate default values while calling this Component